marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/entitiesSidepanel.cy.tsx

```diff
@@ -297,4 +297,32 @@ describe("Test '/Explore' left sidebar", () => {
     entitiesSidebar.getInputSearch().should("have.value", "");
     browsePage.getHCTableRows().should("have.length.greaterThan", 1);
   });
+
+  it("Verify that if we select any base entity, the related entities check is selected with all the related entities", () => {
+    browsePage.getGraphView().click();
+    browsePage.waitForSpinnerToDisappear();
+
+    cy.log("**Selecting All entities**");
+    entitiesSidebar.getBaseEntityDropdown().click("right");
+    entitiesSidebar.selectBaseEntityOption("All Entities");
+    entitiesSidebar.getBaseEntityDropdown().click("right");
+    entitiesSidebar.selectBaseEntityOption("BabyRegistry");
+    cy.wait(2000);
```

Comment:
Instead of an explicit wait, let's wait for all request to be done:

![image](https://github.com/marklogic/marklogic-data-hub/assets/126084579/33c33571-affd-4c4d-8f1b-f7cb8b7cc70f)

You can intercept the last request and wait to that specific request, or just use the cy.waitForAsynRequest();